packages/rx_bloc_cli/mason_templates/bricks/feature_cicd_fastlane/__brick__/{{#cicd_github}}.github{{/cicd_github}}/workflows/fastlane_ios_custom_build_and_deploy.yaml

```diff
@@ -53,9 +53,6 @@ jobs:
         with:
           flutter-version: ${{ inputs.flutter_version }}
           channel: "stable"
-          cache: true
-          cache-key: "flutter-:os:-:channel:-:version:-:arch:-:hash:-${{ hashFiles('**/pubspec.lock') }}"
-          cache-path: "${{ runner.tool_cache }}/flutter/:channel:-:version:-:arch:"
```

Comment:
Please do not remove any rx_bloc_cli template code unless there is a reason for it. Also make sure to always double check what files you are modifying. You're modifying rx_bloc_cli template files in a PR that has to do with the todo example app.

Reply:
After some look at the code history, it turns out this came from develop. Those changes were made intentionally so closing this.